### DISCUSSION TITLE
Multiple databases with a single sql server connection being referenced using DAB

### DISCUSSION THREAD

**Discussion by dehlers-cts** [Azure/data-api-builder#2386] [General]
I have a connection to a single SQL Server with two databases. I should be able to query the tables from different databases. there is a Justice.dbo.Citation table and a ELF.ELFFam.Department. Even though the connection has access to both the tables, I am unable to reference them without getting an error. I would expect something like this: 

 "Department": {
      "source": {
        "object": "ELF.ELFFam.Department",
        "type": "table"
      },

It throws an error. Is there something simple I am missing?

**Comment by seantleonard** [comment]
Hi @dehlers-cts , You will need to utilize DAB's "multi data source" configuration options to get this to work: https://learn.microsoft.com/en-us/azure/data-api-builder/reference-configuration#data-source-files

**Reply by dehlers-cts** [reply]
I appreciate the feedback. I should be good now.

**Comment by sezal98** [comment]
Resolving now
